gnome-initial-setup/pages/language/cc-common-language.c

```diff
@@ -285,13 +285,59 @@ insert_user_languages (GHashTable *ht)
         }
 }
 
-GHashTable *
-cc_common_language_get_initial_languages (void)
+#define VENDOR_LANGUAGE_GROUP "Language"
+#define VENDOR_LANGUAGE_INITIAL_LANGUAGES_KEY "initial_languages"
+
+static gboolean
+insert_vendor_languages (GHashTable *ht)
 {
-        GHashTable *ht;
+        g_autoptr(GKeyFile) keyfile = NULL;
+        g_autoptr(GError) error = NULL;
+        g_autofree char **languages = NULL;
```

Comment:
I think this is the wrong free macro, shouldn't we have g_autoptr(GStrv)? g_key_file_get_string_list is transfer_full and insert_language copies the language it's given.